addons/l10n_cl/models/res_partner.py

```diff
@@ -21,3 +22,32 @@ class ResPartner(models.Model):
              '2 - Fees Receipt Issuer (Applies to suppliers who issue fees receipt)\n'
              '3 - End consumer (only receipts)\n'
              '4 - Foreigner')
+
+    def _format_vat_cl(self, values):
+        identification_types = [self.env.ref('l10n_latam_base.it_vat').id, self.env.ref('l10n_cl.it_RUT').id,
+                                self.env.ref('l10n_cl.it_RUN').id]
+        country_id = self.env.ref('base.cl').id
```

Comment:
could also do a boolean country_is_chile

Reply:
In 1528906cf6c54702ee6290be83751f211b1a6903

I added two booleans; one for the company in the environment and the other one for the partner:

https://github.com/bmya/odoo/blob/1528906cf6c54702ee6290be83751f211b1a6903/addons/l10n_cl/models/res_partner.py#L26-L36